negotiation/neural/generator.py

```diff
@@ -102,6 +102,17 @@ def unbottle(m):
         enc_states, memory_bank = self.model.encoder(encoder_inputs, lengths)
         dec_states = self.model.decoder.init_decoder_state(
                                         encoder_inputs, memory_bank, enc_states)
+
+        if hasattr(batch, 'prev_turns'):
+            item_title = batch.item_title
+            previous_turns = batch.prev_turns
+
+            _, item_memory_bank = self.model.cbow_embedder(item_title)
+            _, prev_memory_bank = self.model.cbow_embedder(previous_turns)
+            enc_memory_bank = memory_bank
+
+            memory_bank = [enc_memory_bank, prev_memory_bank.transpose(0,1)]
+            predict_with_context = True
```

Comment:
don't do adhoc option here. use model option to decide if we're using context, e.g. check num_context